bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gnirs/EdCompInstGNIRS.java

```diff
@@ -76,6 +76,19 @@ public EdCompInstGNIRS() {
         _w.disperser.setChoices(SpTypeUtil.getFormattedDisplayValueAndDescriptions(Disperser.class));
         _w.centralWavelength.setChoices(_getDefaultWavelengths());
 
+        //TODO: Add filter and acqMirror to GNIRS instrument component
```

Comment:
In some places I commented out some code which wasn't ready for this merge yet, but I'd like to use it later. So if it's ok, I'd prefer to leave it there...